src/Shared/CoreCLRAssemblyLoader.cs

```diff
@@ -114,5 +143,15 @@ private Assembly LoadAndCache(string fullPath)
 
             return assembly;
         }
+
+        private bool IsAssemblyAlreadyLoaded(string path)
+        {
+            return _pathsToAssemblies.ContainsKey(path);
+        }
+
+        private bool FileMatchesAssemblyName(string path, string assemblySimpleName)
```

Comment:
Won't be needed after the above comment is resolved. It was only called in one place